### ISSUE TITLE
Ionic 2 - Toast causes errors when displayed and user is navigating in parallel

### ISSUE THREAD

**Issue by DavidTalamona** [driftyco/ionic#8191]
#### Short description of the problem:
Two pages. 2nd does an ajax call then displays a toast and pops itself out of the page stack.
Back on page 1 the user presses the button leading to page 2 again and the following error appears:
"Expression has changed after it was checked".

I assumed the errors cause is the not anymore existing page that called it but when I create the toast via a static provider the same error is thrown.

**Which Ionic Version?** 2.0.0-beta.37

#### Plunker that shows an example of your issue
I cannot get the punker to work. As soon as I import the ToastController it crashes BUT I did create an example with the ToastController commented out. Feel free to uncomment it and do the correct import.
http://plnkr.co/edit/pVp1Wx5qh2MC5wJOwvrI?p=preview

Navigate to page 2, click the button and to to page 2 again as long as the toast is displayed.

**Run `ionic info` from terminal/cmd prompt:**
Cordova CLI: 6.3.1
Ionic CLI Version: 2.0.0-beta.37
Ionic App Lib Version: 2.0.0-beta.20
OS:
Node Version: v4.5.0

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us! This issue has already been fixed and will be in the beta.12 release. Thanks for using Ionic!

**Comment by DavidTalamona** [comment]
I saw all the other posted issues about this specific error with the Toast but always in combination with other stuff like the DateTimePicker.
I also saw your "fixed in beta.12" comment BUT since I'm using beta.37 which is way beyond beta.12 I thought this must be a new issue.
Could you please explain why it's still not fixed in 37? Am I missing something about the version numbers? Thanks.

**Comment by DavidTalamona** [comment]
Never mind, I just did the "ionic info" again and not it displays a "Framework Version" which did not show up yesterday.

Cordova CLI: 6.3.1
**Ionic Framework Version: 2.0.0-beta.11**
Ionic CLI Version: 2.0.0-beta.37
Ionic App Lib Version: 2.0.0-beta.20
OS:
Node Version: v4.5.0

**Comment by ramonornela** [comment]
@DavidTalamona Wait beta.12 only version **nightly**, they are with the features and fix, big changes to come.

Thanks
